_data/navigation.yml

```diff
@@ -656,6 +656,23 @@ reference:
       url: /reference/pipeline/stages/
     - title: Pipeline Templates
       url: /reference/pipeline/templates/
+  - title: Managed Delivery
```

Comment:
I can't test this locally apparently because jekyll doesn't like me or I'm doing something very wrong. So, YOLO.